components/app.js

```diff
@@ -0,0 +1,33 @@
+// Primary class
+class App {
+  constructor(gradeTable) {
+    this.gradeTable = gradeTable;
+    this.handleGetGradesError = this.handleGetGradesError.bind(this);
+    this.handleGetGradesSuccess = this.handleGetGradesSuccess.bind(this);
+  }
+
+  handleGetGradesError(error) {
+    console.error(error);
+    console.log("Error");
+  }
+
+  handleGetGradesSuccess(grades) {
+    console.log("log grades", grades);
+    console.log("Success");
```

Comment:
Make sure to remove `console.log`s once you are done with them and before making a pull request to master. Production code shouldn't have `console.log`s in it, the only exception being `console.error` for errors.